### PR TITLE
Improvement/update to latest core

### PR DESCRIPTION
**Pull request by rfreitas**
Changes were done to the sdk ui in order to conform with the changes done to the onfido-sdk-core.

### CODE REVIEW

File: src/components/Capture/index.js [hunk 1]
```diff
@@ -47,19 +50,20 @@ export default class Capture extends Component {
 
     const { actions, socket, documentType } = this.props
     const methods = {
-      'document': (payload) => {
-        payload.isValid = false
-        payload.documentType = documentType
-        socket.sendMessage(JSON.stringify(payload))
-        actions.documentCapture(payload)
+      'document': (methodPayload) => {
+        var data = methodPayload.data;
```
Comment (stephencookdev) [inline src/components/Capture/index.js]: [minor] You're scattering some semicolons in places. The current code standard for this project is no semicolons

File: src/components/Confirm/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ const Previews = (props) =>  {
       <Capture {...props} captures={props.captures} />
       <div className='onfido-actions'>
         <button
-          onClick={() => props.action(props.captures)}
+          onClick={() => {props.action(props.method);}}
```
Comment (stephencookdev) [inline src/components/Confirm/index.js]: You shouldn't need this wrapped in a block, should you?

File: src/components/Capture/index.js (hunk 1 above)
Comment (stephencookdev) [inline src/components/Capture/index.js]: There's also no real reason to use `var` in ES6+ - `let` and `const` are more explicit, this should be `const` here

**Comment by stephencookdev** [comment]
👍  LGTM mostly, all of my comments are pretty minor

File: src/components/Capture/index.js (hunk 1 above)
Comment (rfreitas) [inline src/components/Capture/index.js]: noted, I'll switch to const

Comment (rfreitas) [inline src/components/Capture/index.js]: any reason why we are going with no semicolon? there are some cases where no semicolon can cause issues.

File: src/components/Confirm/index.js (hunk 2 above)
Comment (rfreitas) [inline src/components/Confirm/index.js]: you are right, it was added in testing when I put a console.log inside

**Comment by rfreitas** [comment]
syntax issues have been fixed, so PR will be merged

**Pull request merged**